docs/docsite/rst/community/collection_development_process.rst

```diff
@@ -71,35 +71,83 @@ Creating a changelog fragment
+
 
 **deprecated_features**
-  Features that have been deprecated and are scheduled for removal in a future release. Displayed in both the changelogs and the :ref:`Porting Guides <porting_guides>`.
+  Features that have been deprecated and are scheduled for removal in a future release. Use past tense and include an alternative, where available for what is being deprecated.. Displayed in both the changelogs and the :ref:`Porting Guides <porting_guides>`.
+
+  .. code-block:: yaml
+
+    deprecated_features:
+      - mail callback plugin - not specifying ``sender`` is deprecated and will be disallowed in ``community.general`` 6.0.0 (https://github.com/ansible-collections/community.general/pull/4140).
+
 
 **removed_features**
-  Features that were previously deprecated and are now removed. Displayed in both the changelogs and the :ref:`Porting Guides <porting_guides>`.
+  Features that were previously deprecated and are now removed. Use past tense and include an alternative, where available for what is being deprecated. Displayed in both the changelogs and the :ref:`Porting Guides <porting_guides>`.
+
+  .. code-block:: yaml
+
+    removed_features:
+      - acme_account_facts - the deprecated redirect has been removed. Use ``community.crypto.acme_account_info`` instead (https://github.com/ansible-collections/community.crypto/pull/290).
+
 
 **security_fixes**
-  Fixes that address CVEs or resolve security concerns. Include links to CVE information.
+  Fixes that address CVEs or resolve security concerns. MUST use security_fixes for any CVEs. Use present tense. Include links to CVE information.
+
+  .. code-block:: yaml
+
+    security_fixes:
+      - win_psexec - ensure password is masked in ``psexec_``command return result (https://github.com/ansible-collections/community.windows/issues/43).
+
 
 **bugfixes**
-  Fixes that resolve issues.
+  Fixes that resolve issues. SHOULD not be used for minor enhancements (use ``minor_change`` instead). Use past tense to describe the problem and present tense to describe the fix.
+
+  .. code-block:: yaml
+
+    bugfixes:
+      - apt_repository - fix crash caused by ``cache.update()`` raising an ``IOError`` due to a timeout in ``apt update`` (https://github.com/ansible/ansible/issues/51995).
+
 
 **known_issues**
-  Known issues that are currently not fixed or will not be fixed.
+  Known issues that are currently not fixed or will not be fixed. Use present tense and where available, use imperative tense for a workaround.
+
+  .. code-block:: yaml
+
+    known_issues:
+      - idrac_user - module may error out with the message unable to perform the import or export operation because there are pending attribute changes or a configuration job is in progress. Wait for the job to complete and run the task again.(https://github.com/dell/dellemc-openmanage-ansible-modules/pull/303).
```

Comment:
Not sure which part of this is the error message, should we call it out? 
```suggestion
      - idrac_user - module may error out with the message ``unable to perform the import or export operation`` because there are pending attribute changes or a configuration job is in progress. Wait for the job to complete and run the task again.(https://github.com/dell/dellemc-openmanage-ansible-modules/pull/303).
```